### DISCUSSION TITLE
Система передачи изображений

### DISCUSSION THREAD

**Discussion by VigersRay** [space-sunrise/space-station-14#527] [Ideas]
Нужно написать систему на подобии ттса но не для голоса а для изображений.
Использовать можно будет для таких механик:
- фотографии (как бумажки но на них вместо текста пичка).
- прикрепление изображений в новостях.
- прикрепление изображений в месенджере.

В целом логика проста, каждый клиент может отправить на сервер фото в байтах, там оно сохраняется и ему присваевается айдишник, а все остальные клиенты запрашивают эту пикчу и кешируют её в памяти. 

Если кто хочет взяться - делайте, я возьмусь не раньше зимы.

**Comment by andreySS14** [comment]
хотелось бы
